### ISSUE TITLE
Scripts vs. scripts

### ISSUE THREAD

**Issue by bantu** [IbcAlpha/IBC#32]
There are two folders in this repository:
https://github.com/IbcAlpha/IBC/tree/master/resources/Scripts
https://github.com/IbcAlpha/IBC/tree/master/resources/scripts

This will probably lead to problems when checked out on a case-insensitive filesystem.

Suggesting to drop one of the two.

**Comment by rlktradewright** [comment]
Thanks for spotting that.

Actually it doesn't cause problems on a case-insensitive filesystem: I do the development on Windows, and because it works fine I just hadn't noticed. 

I think I'll have to clone it to my Ubuntu system to fix it. (Some time...)

**Comment by bantu** [comment]
@rlktradewright The fact that you're not seeing any issues, does not imply that there are no issues and won't be any issues in the future.

Feel free to just tell me how to resolve this. Here is the diff.
```diff
diff -ur scripts/ Scripts/
Only in scripts/: displaybannerandlaunch.sh
Only in Scripts/: DisplayBannerAndLaunch.sh
Only in scripts/: ibcstart.sh
diff -ur scripts/README.txt Scripts/README.txt
--- scripts/README.txt	2018-11-29 17:44:58.632814802 +0100
+++ Scripts/README.txt	2018-11-29 17:44:58.628814769 +0100
@@ -1,3 +1,3 @@
-This folder contains scripts that should never need to be amended by the end user.
-
-Please DO NOT CHANGE THESE FILES unless you are sure you know what you are doing!!!
+This folder contains scripts that should never need to be amended by the end user.
+
+Please DO NOT CHANGE THESE FILES unless you are sure you know what you are doing!!!
diff -ur scripts/SendStopCommand.vbs Scripts/SendStopCommand.vbs
--- scripts/SendStopCommand.vbs	2018-11-29 17:44:58.632814802 +0100
+++ Scripts/SendStopCommand.vbs	2018-11-29 17:44:58.628814769 +0100
@@ -1,18 +1,18 @@
-' allow time for telnet session to start and connect
-WScript.sleep 200 
-
-set OBJECT=WScript.CreateObject("WScript.Shell")
-
-' STOP IBC
-OBJECT.SendKeys "STOP{ENTER}" 
-
-WScript.sleep 50 
-
-' Disconnect from IBC
-OBJECT.SendKeys "EXIT{ENTER}" 
-
-WScript.sleep 50 
-
-' close telnet window
-OBJECT.SendKeys "q{ENTER}" 
-
+' allow time for telnet session to start and connect
+WScript.sleep 200 
+
+set OBJECT=WScript.CreateObject("WScript.Shell")
+
+' STOP IBC
+OBJECT.SendKeys "STOP{ENTER}" 
+
+WScript.sleep 50 
+
+' Disconnect from IBC
+OBJECT.SendKeys "EXIT{ENTER}" 
+
+WScript.sleep 50 
+
+' close telnet window
+OBJECT.SendKeys "q{ENTER}" 
+
```

**Comment by bantu** [comment]
There is also config.ini vs Config.ini

**Comment by rlktradewright** [comment]
Well, I'm not having issues and no-one else has reported issues, so I presume it's not causing anyone any problems at present. So as far as I'm concerned, it's not urgent and therefore I'll get round to fixing it when I have the time to do so.

If you want to fix it feel free. I imagine that the way to do this is to clone the repository to a case-sensitive file-system, such as a Linux system, which will give you two different folders, scripts and Scripts. Then move all the files from Scripts to scripts.

Also copy Config.ini to config.ini, and delete Config.ini.

Then submit a PR with the changes.

I imagine this will be straightforward, but things have a habit of biting you in the bum if you haven't thought them through, which is why I wanted to leave it till I've got time to be sure it'll be alright. But as I say, if you want to give it a try, feel free.

**Comment by rlktradewright** [comment]
This issue was addressed in commit ec8ebbec.

If you have cloned of this repository on a case-sensitive operating system (Linux, macOS, FreeBSD etc), then you can pull these changes in the normal fashion, and the problem will have been corrected, ie the duplicate files with the incorrectly spelled paths/ filenames will have been deleted.

So if you're using a case-sensitive operating system, the rest of this post is not relevant for you.

However, if you've cloned to a case-insensitive operating systems (eg Windows), an additional step is required. Pulling this commit to your clone will result in all the right files being in the right places. But due to the nature of the change, delete updates for the relevant files also will be recorded (but not staged) in Git. **Do not commit those changes**, since this will wipe out the files. These spurious delete updates need to be removed. The relevant files are:

Resources\Config.ini
Resources\Scripts\DisplayBannerAndLaunch.bat
Resources\Scripts\displaybannerandlaunch.sh
Resources\Scripts\getDayOfWeek.bat
Resources\Scripts\ibcstart.sh
Resources\Scripts\README.txt
Resources\Scripts\SendStopCommand.vbs
Resources\Scripts\StartIBC.bat

Here's what to do to delete them:

* if you use a visual Git client such as GitHub Desktop, you'll be able to see these changes directly: simply ask for the changes to be undone. For example in GitHub Desktop's 'Changes' tab, select the files, right-click, and select 'discard n selected changes...'

* If you use the command line, you can see the changes using `git status`. If you haven't  made any other changes since pulling ec8ebbec, the simplest way to remove them is to reset HEAD to point to commit ec8ebbec, thus:
```
git reset --hard ec8ebbec
```
* However this command will also undo any other changes made since you pulled commit ec8ebbec: if you have made such changes that you want to keep, an alternative way to remove the unwanted delete modifications is to issue this command for each of the relevant files:
```
git checkout -- <filename>
```
I apologise to Windows users for the inconvenience of having to do this, but I wasn't able to find a way of correcting the problem on Linux/macOS/etc without also having this undesirable effect on Windows.

**Comment by bantu** [comment]
@rlktradewright I think this can be closed, no?
